show/main.py

```diff
@@ -1470,12 +1475,33 @@ def ztp(status, verbose):
        cmd = cmd + " --verbose"
     run_command(cmd, display_cmd=verbose)
 
+#
+# Install platform extensions for show commands
+# extends support for cli and platform
+#
+def install_extensions():
+    # Extend parsers as desired
+    try:
+        extensions = sonic_platform.cli.show.extensions() or {}
+    except:
+        extensions = {}
+
+    groups = {
+        'cli': cli,
+        'platform': platform.platform,
+    }
+
+    for key,root in groups.items():
+        for cmd in extensions.get(key, []):
+            root.add_command(cmd)
 
 # Load plugins and register them
 helper = util_base.UtilHelper()
 for plugin in helper.load_plugins(plugins):
     helper.register_plugin(plugin, cli)
 
+# invoke method for installing extensions
+install_extensions()
```

Comment:
Would this cause inconsistent platform CLI's across SONiC running in different vendor devices ?  As as i know we have common platform CLI's and platform API support is added via platform plugins. @jleveque to comment as well.

Reply:
as per review comments, the code is restructured to the below (along with PR1577)

the underlying show_platform/cli/show.py will be moved under the respective plugin path as follows:

   sonic-utilities/show/plugins/cisco-8000.py 
Similar future enhancements for config and clear options will be as:

      sonic-utilities/config/plugins/cisco-8000.py
      sonic-utilities/clear/plugins/cisco-8000.py

Reply:
> Would this cause inconsistent platform CLI's across SONiC running in different vendor devices ? As as i know we have common platform CLI's and platform API support is added via platform plugins. @jleveque to comment as well.

the tests failed after changes in dbp arp and vlan tests which has no intersection to the code changes that were made. Is it possible to kick off a rerun of the tests (i dont have privileges to do so).